test/repo.js

```diff
@@ -21,6 +22,18 @@ exports.get = {
 exports.fetch = {
 	setUp: function( done ) {
 		this.repo = new Repo( "test-repo" );
+
+		this.retry = util.retry;
+		util.retry = function( method ) {
+			return method();
+		};
```

Comment:
I wasn't too sure what to do about this. I don't want to convolute the tests for fetching with retry logic, so I just set `util.retry()` to be a direct pass-through. I suppose a better option might be to explicitly check for `util.retry()` being invoked where we expect it and then having that just do the pass-through.